pkg/reconciler/v1alpha1/clusteringress/resources/virtual_service.go

```diff
@@ -118,8 +139,9 @@ func makeVirtualServiceRoute(hosts []string, http *v1alpha1.HTTPClusterIngressPa
 			Attempts:      http.Retries.Attempts,
 			PerTryTimeout: http.Retries.PerTryTimeout.Duration.String(),
 		},
-		AppendHeaders:    http.AppendHeaders,
-		WebsocketUpgrade: true,
+		DeprecatedAppendHeaders: http.AppendHeaders,
```

Comment:
Please add a TODO here to explain the "Deprecated" prefix usage.

Reply:
Done